examples/modus_toolbox/README.md

```diff
@@ -22,7 +22,7 @@ This example supports all Golioth services, including Over-the-Air firmware
 updates:
 
 * LightDB: https://docs.golioth.io/cloud/services/lightdb
-* LightDB Stream: https://docs.golioth.io/cloud/services/lightdb-stream
+* Stream: https://docs.golioth.io/cloud/services/stream
```

Comment:
Let's go ahead and make this update as the page already exists and will be what we want to be pointing to shortly.
```suggestion
* Stream: https://docs.golioth.io/data-routing
```